src/main/java/styluscar/client/ClientResource.java

```diff
@@ -5,32 +5,34 @@
 
 import javax.inject.Inject;
 import javax.transaction.Transactional;
-import javax.ws.rs.GET;
-import javax.ws.rs.POST;
-import javax.ws.rs.PUT;
-import javax.ws.rs.Path;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import java.awt.*;
 import java.util.List;
 
 /**
  * Endpoints relacionados aos clientes
  */
+@Produces("application/jason")
+@Consumes("application/jason")
 @Path("client")
 public class ClientResource {
     @Inject
     ClientService service;
 
     @GET
     @Path("/clients")
+    @Produces(MediaType.APPLICATION_JSON)
     public List<Client> getAllClients() {
         try {
-            return this.service.findAllClient();
+           List<Client> clients = this.service.findAllClient();
+           return clients;
```

Comment:
You can use `return this.service.findAllCliente();`

Reply:
done